pkg/ipam/metrics/metrics.go

```diff
@@ -50,74 +48,60 @@ func NewPrometheusMetrics(namespace string, registry *prometheus.Registry) *prom
 
 	m.IPsAllocated = prometheus.NewGaugeVec(prometheus.GaugeOpts{
 		Namespace: namespace,
-		Subsystem: eniSubsystem,
+		Subsystem: ipamSubsystem,
 		Name:      "ips",
 		Help:      "Number of IPs allocated",
 	}, []string{"type"})
 
 	m.AllocateIpOps = prometheus.NewCounterVec(prometheus.CounterOpts{
 		Namespace: namespace,
-		Subsystem: eniSubsystem,
+		Subsystem: ipamSubsystem,
 		Name:      "allocation_ops",
 		Help:      "Number of IP allocation operations",
 	}, []string{"subnetId"})
 
 	m.ReleaseIpOps = prometheus.NewCounterVec(prometheus.CounterOpts{
 		Namespace: namespace,
-		Subsystem: eniSubsystem,
+		Subsystem: ipamSubsystem,
 		Name:      "release_ops",
 		Help:      "Number of IP release operations",
 	}, []string{"subnetId"})
 
 	m.AllocateEniOps = prometheus.NewCounterVec(prometheus.CounterOpts{
 		Namespace: namespace,
-		Subsystem: eniSubsystem,
+		Subsystem: ipamSubsystem,
 		Name:      "interface_creation_ops",
 		Help:      "Number of ENIs allocated",
```

Comment:
Nit: should this say `Number of interfaces allocated`?

```suggestion
		Help:      "Number of interfaces allocated",
```

Probably also rename `AllocateEniOps` to `AllocateInterfaceOps`.

Reply:
Just realized that the PR was merged already. I sent #10406 with the suggested fix.